### ISSUE TITLE
[Accessibility] Payment methods should be radio options

### ISSUE THREAD

**Issue by gazjoy** [Adyen/adyen-web#1695]
**Describe the bug**
As part of [WCAG 2.1 - 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) the payment methods should give meaning of their relationship to each other. This is because they are separate buttons with no relationship. This is important as giving the methods a mutually exclusive relationship means a screen reader user will understand there are options for the method of payment. We only want to give users one option but also describe to the user that options are available. input type="radio" does this. At the moment the only way for screen reader users to understand options are available is to jump forward and backwards in the document. They could easily miss the fact that they could pay with PayPal or any other option.

**To Reproduce**
1. Go to payment step with screen reader
2. Listen to description of payment step

**Expected behavior**
Users should be able to understand they can use their credit card or another means. When navigated to by a screen reader I would expect to hear:

- Pay by Credit card - radio button 1 of 2
- Pay with PayPal - radio button 2 of 2

**Screenshots**
![image](https://user-images.githubusercontent.com/628064/185103573-8d7342fa-6d7c-4da1-86aa-b72b5b71fd97.png)
![image](https://user-images.githubusercontent.com/628064/185103697-1c288a28-844c-4119-97ad-ea54d1c8b146.png)
![image](https://user-images.githubusercontent.com/628064/185103764-2ef67cbc-7fe3-4b56-a507-33f1be800600.png)

**Desktop (please complete the following information):**
- OS: macOS Monterey 12.5
- Browser: Safari 15.6
- Screen reader: VoiceOver

**Comment by ribeiroguilherme** [comment]
Hey @gazjoy ,

Thanks for getting back to us with these accessibility issues. Can you also share what version of the SDK are you using during your tests?

**Comment by sponglord** [comment]
And, as mentioned in https://github.com/Adyen/adyen-web/issues/1691#issuecomment-1220451031, this is also something we expect to be reported on in our ADA / WCAG accessibility review

**Comment by gazjoy** [comment]
> Hey @gazjoy ,
> 
> Thanks for getting back to us with these accessibility issues. Can you also share what version of the SDK are you using during your tests?

version is: "@adyen/adyen-web": "5.22.0"
implementation is: drop-in

**Comment by gazjoy** [comment]
@m1aw Hi is there any update on when these will be made into radios? Thank you.

**Comment by m1aw** [comment]
Hey @gazjoy yeah. We are going forward with this in v6. We do not have a release date yet, but I we are thinking in the next few months.

**Comment by pedrogarmin** [comment]
Hi @m1aw 
Do you have any release date for this?

**Comment by m1aw** [comment]
We are now targeting the first few months next year, tho this is a concrete date.

One question, does the current solution not solve the problem? Because the list is correctly marked with the appropriate role.

**Comment by m1aw** [comment]
The major release, v6.0.0, was launched today!
See the documentation here:

[release notes](https://docs.adyen.com/online-payments/release-notes/#releaseNote=2024-07-24-web-componentsdrop-in-6.0.0)
[migration guide](https://docs.adyen.com/online-payments/update-your-integration/migrate-to-web-v6/)

In this version we made the list into radiogroup/radio buttons. The mark up should be correct. Closing this issue.
